media/js/base/global.js

```diff
@@ -34,15 +34,19 @@ function init_download_links() {
 }
 
 function update_download_text_for_old_fx() {
-    // if using an out of date firefox
-    if (isFirefox() && !isFirefoxUpToDate()) {
-        // look at each button to see if it's set to check for old firefox
-        $('.download-button').each(function() {
-            var $button = $(this);
-
-            if ($button.hasClass('download-button-check-old-fx')) {
-                // replace subtitle copy
-                $button.find('.download-subtitle').text(window.trans('global-update-firefox'));
+    if (isFirefox()) {
+        window.getFirefoxDetails(function(data) {
```

Comment:
This is firing a UITour event for every page on bedrock. Looking at the code however, it seems as though there is only a single occurrence of this particular button style on the [`/plugincheck` page](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/plugincheck.html#L100). This also means the /plugincheck page is firing this custom event twice to retrieve the same information. 

Instead of calling this function automatically for every page on this site, should we instead omit the up-to-date logic here and call it explicitly from `check-plugins.js`?